client/src/ui/components/friends/ChatWindow.tsx

```diff
@@ -19,9 +37,45 @@ const ChatWindow: React.FC<ChatWindowProps> = ({ chat }) => {
         isMobile ? "" : "rounded-2xl border-l-4 border-b-4 border-accent-100"
       }`}
     >
-      {chat.messages.map((message, index) => (
-        <div key={index}>{message.content}</div>
-      ))}
+      <div className="title-container flex flex-row justify-left w-full h-1/6 min-h-20 max-h-24 h-fit">
+        <img
+          src={chat.chatProfilePicture}
+          className="object-contain max-h-20 max-w-20 m-4"
+        ></img>
+        <div className="flex flex-col w-full max-h-20">
+          <div className="flex flex-row w-full items-center justify-center">
+            <Title title={chat.chatName} />
+            <div
+              className={`activity rounded-full h-4 w-4 text-white font-bold flex items-center justify-center ml-2 mt-2 ${
+                chat.members.some((member) => member.active)
+                  ? "bg-green"
+                  : "bg-red"
+              }`}
+            ></div>
+            <div className="flex flex-grow"></div>
+            {isGroupChat && (
+              <button>
+                <FontAwesomeIcon icon={faEdit} className="m-2 mr-4 title" />
+              </button>
+            )}
+          </div>
+        </div>
+      </div>
+      <div className="bg-primary-100 h-1 w-full" />
+      <div className="message-container w-full overflow-auto no-scrollbar">
+        {chat.messages.map((message, index) => {
+          const previousMessage = chat.messages[index - 1];
```

Comment:
What if there's only one message and you try to access `index - 1`

Reply:
Added a quick filler in case there are no messages